src/main.cpp

```diff
@@ -174,6 +176,7 @@ const static struct option long_opt[] =
   {"pidfile",                     required_argument, NULL, PIDFILE},
   {"daemon",                      no_argument,       NULL, DAEMON},
   {"sas-use-signaling-interface", no_argument,       NULL, SAS_USE_SIGNALING_IF},
+  {"support-shared-ifcs",         no_argument,       NULL, SUPPORT_SHARED_IFCS},
```

Comment:
The config option that's passed into Homestead uses dashes, and the config option in the shared config file uses underscores. The init.d file (which I'd forgotten to check in) reads the shared config and sets the Homestead configuration as appropriate